tailwind.config.js

```diff
@@ -67,6 +74,10 @@ module.exports = {
           450: '#3F9AF7',
           '3f': '#3F9AF7',
```

Comment:
Duplicate color

Reply:
Thanks for catching, removed blue-450 and replaced its usage to blue-3f in Header